fe/fe-core/src/main/java/org/apache/doris/statistics/StatisticsCache.java

```diff
@@ -146,6 +147,14 @@ public void preHeat() {
     private void doPreHeat() {
         List<ResultRow> recentStatsUpdatedCols = null;
         long retryTimes = 0;
+        while (!StatisticsUtil.statsTblAvailable()) {
```

Comment:
if the statsTbl is not available for a long time, does doPreHeat block fe start up?

Reply:
no